common/app/conf/switches/MonitoringSwitches.scala

```diff
@@ -10,6 +10,7 @@ trait MonitoringSwitches {
     SwitchGroup.Monitoring,
     "offline-page-view",
     "If this switch is on, views of the offline page will be beaconed to cloudwatch when you come online again",
+    owners = Seq(Owner.withGithub("sndrs")),
     safeState = Off,
     sellByDate = new LocalDate(2016, 6, 3),
```

Comment:
ooh good spot - @sndrs won't be able to do anything with that, we should extend until next wednesdayish!